app/templates/auth/create-user.html

```diff
@@ -0,0 +1,84 @@
+{% extends "_base_page.html" %}
+
+{% block page_title %}Create account – Digital Marketplace{% endblock %}
+
+{% block main_content %}
+
+{% if not token %}
```

Comment:
Is it possible to arrive at this page without a token?  
It seems like in both places this template is called, the token has already been checked.